src/pages/welcome-page/welcome-page.scss

```diff
@@ -1,8 +1,71 @@
 page-welcome {
-  .welcome-message {
-    font-family: GDSTransportWebsite-Bold;
-    text-align: center;
-    padding-top: 3rem;
-    font-size: 3rem;
+  font-weight: bold;
+
+  ion-content {
+    background-image: url('/assets/imgs/welcome-page-background.png');
+    -webkit-background-size: cover;
```

Comment:
The various vendor prefixes here are probably a good candidate for a Sass mixin.